core/domain/suggestion_services.py

```diff
@@ -1098,6 +1098,59 @@ def get_reviewable_translation_suggestions_by_offset(
     return translation_suggestions, next_offset
 
 
+def get_reviewable_translation_suggestions_for_single_exp(
+    user_id: str,
+    opportunity_summary_exp_id: str,
+    language_code: str
```

Comment:
You haven't fixed the docstring to include this argument.

Reply:
Done